### PR TITLE
Ggoryl image features as an option

### PR DESCRIPTION
**Pull request by polquant**
This PR adds possibility to train data on image features or landmarks or both

I seems, that after re-base  I have all commits mixed.

### CODE REVIEW

File: train_config_ggoryl.yml [hunk 1]
```diff
@@ -0,0 +1,93 @@
+general_parameters:
+    data_paths: ["/media/data512/hearai/frames/pjm", "/media/data512/hearai/frames/basic_lexicon", "/media/data512/hearai/frames/galex", "/media/data512/hearai/frames/glex"]
```
Comment (AgaMiko) [inline train_config_ggoryl.yml:2]: is this addition intentional? Maybe could you create a message board for configs on basecamp instead? Let me know what do you think!

File: models/model.py [hunk 2]
```diff
@@ -79,7 +80,7 @@ def __init__(
             self.run = initialize_neptun(tags)
             self.run["parameters"] = {
                 "general_parameters": general_parameters,
-                "train_parameters": general_parameters,
+                "train_parameters": train_parameters,
```
Comment (polquant) [inline models/model.py:83]: small fix

**Comment by polquant** [comment]
> This PR adds possibility to train data on image features or landmarks or both

**Comment by polquant** [comment]
I have created a new PR without  mixed commits from main
